trappy/ftrace.py

```diff
@@ -168,28 +168,21 @@ class definitions list.  Otherwise, register a class to parse
     def __populate_data(self, fin, cls_for_unique_word, window, abs_window):
         """Append to trace data from a txt trace"""
 
-        def contains_unique_word(line, unique_words=cls_for_unique_word.keys()):
-            for unique_word in unique_words:
-                if unique_word in line:
-                    return True
-            return False
-
         actual_trace = itertools.dropwhile(self.trace_hasnt_started(), fin)
         actual_trace = itertools.takewhile(self.trace_hasnt_finished(),
                                            actual_trace)
 
         for line in actual_trace:
-            if not contains_unique_word(line):
-                self.lines += 1
-                continue
+            trace_class = None
```

Comment:
Ok sounds good, I'll just add the cleanup PR ontop of the original one to keep it less confusing.